### ISSUE TITLE
Does boost package support Python 3 at all?

### ISSUE THREAD

**Issue by frol** [conda/conda-recipes#353]
I see that the package is named without `py27` or `py34` prefix and I see only one shared library `libboost_python.so`, which seem to be Python 2.7 only version.

**Comment by ccordoba12** [comment]
Not, boost at the moment only supports Python 2

**Comment by frol** [comment]
@ccordoba12 Do you mean boost package in conda supports only Python 2 or what? I indeed cannot find advertising on the main page of Boost about this, but Fedora, Ubuntu, and Arch Linux certainly have boost-python3 packages.

**Comment by frol** [comment]
I'm also curious from where this `${INCLUDE_PATH}` should come from? https://github.com/conda/conda-recipes/blob/master/boost/build.sh#L39

**Comment by ccordoba12** [comment]
@frol: Yes, I meant that the boost conda package only supports Python 2.

I don't know why that `${INCLUDE_PATH}` is used in the recipe.

**Comment by frol** [comment]
It is complete miracle that this package works at all... I'm repackaging it and will also make separate versions for Python 2 and Python 3.

**Comment by ccordoba12** [comment]
I don't understand what you mean by *utter miracle*. The package is working just fine, and it's used by other projects to build their packages too.

**Comment by frol** [comment]
Boost's build scripts don't care if you pass wrong arguments or it couldn't find python/icu or it doesn't compile at all, they always return exit code 0 no matter how badly it fails. If you just run the conda package build as it is now it will reveal that boost package cannot detect icu (even though it is specified as dependency), `include=` will be empty (it will say error, but will continue anyway). There is actually an option `-q`, which should al least stop build process on the first error, but still you won't receive any error indication via exit code.

The bottom line, you were just lucky that it got built boost with all packages in your case...

**Comment by ccordoba12** [comment]
I used these recipes to build our packages, and they were working fine a couple of months ago. `icu` was detected correctly, I actually added that dependency myself. Reviewing `git log`, I just noticed that I added `INCLUDE_PATH` specifically for that purpose.

But I think I recall `INCLUDE_PATH` was removed in conda-build since then, so I need to change that. That's the problem, thanks for reporting it.

**Comment by frol** [comment]
I got an answer to my question, so I close this issue and will probably make a pull request soon.
